plugins/rhs_plugin/index.js

```diff
@@ -10,15 +10,17 @@ import RHSPlugin from './rhs_plugin.jsx';
 
 function mapStateToProps(state) {
     const rhsPlugins = state.plugins.components.RightHandSidebarComponent;
-    const pluginId = getPluginId(state);
+    const pluginComponentId = getPluginId(state);
 
-    const plugin = rhsPlugins.find((element) => element.id === pluginId);
-    const pluginName = plugin ? plugin.title : '';
-    const icons = plugin ? plugin.icons : [];
+    const pluginComponent = rhsPlugins.find((element) => element.id === pluginComponentId);
+    const pluggableId = pluginComponent ? pluginComponent.id : null;
+    const pluginTitle = pluginComponent ? pluginComponent.title : '';
+    const icons = pluginComponent ? pluginComponent.icons : [];
 
     return {
-        pluggableId: pluginId,
-        title: pluginName,
+        showPluggable: Boolean(pluggableId),
```

Comment:
deciding to keep this despite our pivot in direction as it's better to be explicit in showing/hiding the RHS pane for plugins.